NETPROGRAM/String_Exercise/Exercise01.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.String_Exercise
+{
+    public class Exercise01
+    {
+
+        public string SnakeCase(string s)
+        {
+        
+            for(var i = s.Length - 1 ; i > 0 ; i--)
+            {
+
+                if (Char.IsUpper(s[i]))
+                {
+                    s = s.Insert(i, "_");
```

Comment:
Hàm insert thực chất là nó tạo một string hoàn toàn mới bằng cách

- Tạo 1 char array có độ dài = tổng 2 string
- Copy phần đầu của string vào array
- Copy string cần insert
- Copy phần cuối của string 
- Chuyển char array thành string

Tổng cộng nó tốn rất nhiều thao tác --> không dùng nó trong vòng lặp

Reply:
Done.